src/main/resources/check-your-answers/translation.json

```diff
@@ -30,37 +30,45 @@
         "otherInformation": "List of other documents uploaded",
         "additionalInformation": "Additional information",
         "documentRelevance": "Document relevance"
+      },
+      "values": {
+        "yes": "Yes",
+        "no": "No"
       }
     },
     "cy": {
-      "serviceName": "Appeal to the First-tier Tribunal - welsh ",
-      "section": " - welsh ",
-      "title": "Check your answers before submitting your tribunal form - welsh ",
-      "change": "change - welsh ",
-      "continue": "Accept and send - welsh ",
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "section": "",
+      "title": "Gwiriwch eich atebion cyn cyflwyno eich ffurflen tribiwnlys",
+      "change": "newid",
+      "continue": "Derbyn ac anfon",
       "sectionTitles": {
-        "subjectDetails": "Subject Details - welsh ",
-        "representativeDetails": "Representative's Details - welsh ",
-        "representation": "Representation - welsh ",
-        "documents": "Appeal forms - welsh ",
-        "supportingDocuments": "Supporting documents - welsh ",
-        "otherInformation": "Additional information - welsh "
+        "subjectDetails": "Manylion yr Unigolyn Perthnasol",
+        "representativeDetails": "Manylion y cynrychiolydd",
+        "representation": "Cynrychiolaeth",
+        "documents": "Ffurflenni apelio",
+        "supportingDocuments": "Dogfennau ategol",
+        "otherInformation": "Gwybodaeth ychwanegol"
       },
       "keys": {
-        "fullName": "Full name - welsh ",
-        "dateOfBirth": "Date of birth - welsh ",
-        "emailAddress": "Email address - welsh ",
-        "contactNumber": "Contact number - welsh ",
-        "representationLabel": "Is there a representative named on completed tribunal form? - welsh ",
-        "representativeQualifiedLabel": "Is the named representative legally qualified? - welsh ",
-        "representativeFullName": "Full name - welsh ",
-        "representativeOrganisationName": "Organisation or business name - welsh ",
-        "representativeContactNumber": "Contact number - welsh ",
-        "representativeEmailAddress": "Email address - welsh ",
-        "appealDocuments": "List of forms uploaded (Appeal form) - welsh ",
-        "supportingDocuments": "List of supporting documents uploaded - welsh ",
-        "additionalInformation": "Additional information - welsh ",
-        "documentRelevance": "Document relevance - welsh "
+        "fullName": "Enw llawn ",
+        "dateOfBirth": "Dyddiad geni ",
+        "emailAddress": "Cyfeiriad e-bost ",
```

Comment:
Unwanted spaces at the last can be removed.

Reply:
Good spot, will resolve